airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py

```diff
@@ -47,7 +48,12 @@ class TestSpec(BaseTest):
 
     @staticmethod
     def compute_spec_diff(actual_connector_spec: ConnectorSpecification, previous_connector_spec: ConnectorSpecification):
-        return DeepDiff(previous_connector_spec.dict(), actual_connector_spec.dict(), view="tree", ignore_order=True)
+        return DeepDiff(
+            previous_connector_spec.dict()["connectionSpecification"],
+            actual_connector_spec.dict()["connectionSpecification"],
+            view="tree",
+            ignore_order=True,
+        )
```

Comment:
@pedroslopez I changed this to only compute the diff on the `connectionSpecification` field of the spec. I think this is more robust as this is in this field that we want to perform the validation, not the other technical field that exists and might be changed.